HTAN.model.jsonld

```diff
@@ -2475,9 +2475,7 @@
             },
             "sms:displayName": "Filename",
             "sms:required": "sms:true",
-            "sms:validationRules": [
-                "regex search ^\\S*$"
-            ]
+            "sms:validationRules": []
```

Comment:
Looks like we are out of sync with the main branch here for the JSON-LD. We need to keep the validation rule